example/test/get_dom_node_test.dart

```diff
@@ -41,7 +42,7 @@ class SimpleComponent extends react.Component {
     react.div({}, [
       react.span({"ref": "refToSpan"}, "Test"),
       react.span({}, counter),
-      react.button({"onClick": (_) => (getDOMNode() as HtmlElement).children.first.text = (++counter).toString()},"Increase counter"),
+      react.button({"onClick": (_) => (react_dom.findDOMNode(this) as HtmlElement).children.first.text = (++counter).toString()},"Increase counter"),
```

Comment:
But couldn't we have kept it in React Dart and just made it `react_dom.findDomNode(this)`?